.github/workflows/code-checks.yml

```diff
@@ -17,6 +17,6 @@ jobs:
           python-version: 3.8
       - name: Install mypy
         run: |
-          pip install mypy==0.790
+          grep mypy requirements/test.txt | xargs -0 pip install
```

Comment:
why not normal `pip install -r requirements/test.txt`?

Reply:
To avoid installing everything else which isn't necessary to run mypy.

But this is a micro optimization, if you prefer, we can do that too

Reply:
it is fine, I just overlooked it and thought you install all just one-by-one :]